src/uu/head/src/head.rs

```diff
@@ -129,6 +133,7 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         settings.verbose = true;
     }
 
+
```

Comment:
please run rustfmt on the file. thanks

Reply:
Run & committed, knew I forgot something 👍